src/components/ApplicationDashboard.js

```diff
@@ -375,15 +420,99 @@ const Dashboard = ({
         {/* Hides this option if a user unRSVP'd */}
         {hackerRSVPStatus !== "Un-RSVP'd" && canRSVP && (
           <>
-            <SafeWalkContainer>
+            {/* <SafeWalkContainer>
               <QuestionLabel>Safewalk option</QuestionLabel>
               <Checkbox
                 checked={safewalk}
                 onChange={handleChange}
                 label="If you are planning to walk home alone on campus on the night of the 11th, would you like organizers to accompany you to your destination?"
               />
-            </SafeWalkContainer>
-            <WaiverUpload>
+            </SafeWalkContainer> */}
+
+            <CheckboxContainer>
+              <QuestionLabel>
+                Release of Liability <Required />
+              </QuestionLabel>
+              <P>
+                This waiver allows nwPlus to use any photos or videos taken during the event for
+                promotional purposes.
+              </P>
+              <A bolded color="primary" width="130px" href={WAIVER_LINKS.RELEASE_LIABILITY}>
```

Comment:
Very small issue, but we should make the wavers open in the new tab as sometimes the state doesnt get saved when you click directly.